.travis.yml

```diff
@@ -5,4 +5,4 @@ node_js:
   - '7'
   - '8'
 before_install:
-  - 'if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi'
+  - 'if [[ `npm -v` == 2* ]]; then npm i -g npm@5; fi'
```

Comment:
Or remove node v 4 from the travis build?

Reply:
There's still plenty of services in the wild running on node@4 (I imagine), so I wouldn't be too comfortable with that.

Reply:
fair enough